dhcp/config.ml

```diff
@@ -0,0 +1,21 @@
+open Mirage
+
+let main = foreign "Unikernel.Main" (console @-> kv_ro @-> network @-> clock @-> job)
+
+let disk = crunch "files"
+
+(* TODO: workaround a bug in the command-line tool by adding Clock
+   for Unix (this is pulled in as an implicit dependency in Xen) *)
```

Comment:
On Thursday, 12 November 2015, Gabriel Radanne <notifications@github.com>
wrote:

> In dhcp/config.ml
> <https://github.com/mirage/mirage-skeleton/pull/105#discussion_r44648033>:
>
> > @@ -0,0 +1,21 @@
> > +open Mirage
> > +
> > +let main = foreign "Unikernel.Main" (console @-> kv_ro @-> network @-> clock @-> job)
> > +
> > +let disk = crunch "files"
> > +
> > +(* TODO: workaround a bug in the command-line tool by adding Clock
> > +   for Unix (this is pulled in as an implicit dependency in Xen) *)
>
> This is ... ancient. I think @haesbaert <https://github.com/haesbaert>
> copied the config.ml from stack, where this was present. It's not valid
> anymore.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mirage/mirage-skeleton/pull/105/files#r44648033>.
>
Correct, I copied over from the ethif skeleton.
I will clean it up.